### PR TITLE
Fixed eye wear overriding see_invisible when it shouldn't.

### PR DESCRIPTION
**Pull request by xxalpha**
Fixes https://github.com/tgstation/-tg-station/issues/11733

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -678,7 +678,8 @@
 				var/obj/item/clothing/glasses/G = H.glasses
 				H.sight |= G.vision_flags
 				H.see_in_dark = G.darkness_view
-				H.see_invisible = G.invis_view
+				if(G.invis_view < H.see_invisible)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species.dm]: wait... isn't this the wrong way round?

"if the glasses see invisibility is less than the human's see invisibility, override the human's with that of the glasses"

Comment (xxalpha) [inline code/modules/mob/living/carbon/human/species.dm]: dicks

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species.dm]: Just make it ```H.see_invisible = max(H.see_invisible,G.see_invisible)```, that way they always have the most.

**Comment by Supermichael777** [comment]
this should also fix some of the bugs with sling thralls too.

**Comment by xxalpha** [comment]
I've no idea what that means.

**Comment by Supermichael777** [comment]
they had an identical issue where wearing glasses turned off their night vision.

**Pull request merged**